db/versions/0105-migration.sql

```diff
@@ -0,0 +1,15 @@
+begin
+
+  CREATE TABLE audit_history (
+    entity_id text not null,
+    entity_type text not null,
+    client_id text not null,
+    action_type text not null,
+    created timestamptz not null
+  );
+
+  ALTER TABLE audit_history add primary key (entity_id, entity_type, created);
+
+  GRANT select, insert, update, delete ON audit_history to $db_user_prefix$_auth;
```

Comment:
since hooks is using this, we'll need to grant something to the `_hooks` user. 
Plus I think we can use just a subset of permissions.

`auth` should only `select, insert, delete` (udpates should not be enabled at all as we want to ensure that records are not being changed, but we need to keep `delete`  for the expire job)
`hooks` - only `insert` 
`secrets` - only `insert`